pandas/conftest.py

```diff
@@ -79,6 +79,14 @@
     # Import "zoneinfo" could not be resolved (reportMissingImports)
     import zoneinfo  # type: ignore[no-redef]
 
+    # Although zoneinfo can be imported in Py39, it is effectively
+    # "not available" without tzdata/IANA tz data.
+    # We will set zoneinfo to not found in this case
+    try:
+        utc_zoneinfo = zoneinfo.ZoneInfo("UTC")
```

Comment:
```suggestion
        zoneinfo.ZoneInfo("UTC")
```

Just so it's clear that `utc_zoneinfo` isn't used since this is just a smoke test